src/actions/authActions.js

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+
+const config = {
+  headers: {
+    "content-type": "application/json",
+  },
+};
```

Comment:
It's might not require here

Reply:
Yes, you are right -  i have added to make sure that my content in post request is in json always.. that the reason i have added.